resolver/resolver.go

```diff
@@ -63,16 +64,18 @@ func Get(scheme string) Builder {
 }
 
 // SetDefaultScheme sets the default scheme that will be used. The default
-// default scheme is "passthrough".
+// scheme is "passthrough".
```

Comment:
"default default" was not actually a typo.  But maybe would be clearer if it said "the default scheme is initially set to ..." or something like that.